workload/aspnetapp.yaml

```diff
@@ -95,40 +95,33 @@ spec:
     port: 80
     targetPort: 8080
 ---
-apiVersion: networking.k8s.io/v1
-kind: Ingress
+apiVersion: traefik.containo.us/v1alpha1
```

Comment:
Ahh, moving to CRD vs native Ingress.  I like the fact that the baseline doesn't, but this does.  That shows, once again, how you materalize your cluster is really based on your NEEDS, but it can still all be founded on the baseline. (You can auto-resolve this)

Reply:
<3